File: spring-cloud/service-discovery-client/src/main/java/cn/elmi/sample/Application.java
package cn.elmi.sample;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

/**
 * @version 1.0
 * @authur Arthur
 */
@EnableEurekaClient
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: spring-cloud/feign-consumer/api-definition/src/main/kotlin/cn/elmi/sample/feign/Application.kt
package cn.elmi.sample.feign

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients

@EnableFeignClients("cn.elmi.sample.feign.service")
// @EnableEurekaClient
@SpringBootApplication
class Application

fun main() {
    runApplication<Application>()
}